js/mixins/uploader.js

```diff
@@ -30,8 +55,10 @@ define(['api', 'exports?qq!fineuploader'], function(API, qq) {
                     onUpload: this.on_upload,
                     onSubmitted: this.on_submit,
                     onProgress: this.on_progress,
-                    onComplete: this.on_complete
+                    onComplete: this.on_complete,
+                    onError: this.on_error,
```

Comment:
Yes, it's allowed in ES6, or at least it's rewritten by babel.
Having it produce easier to read diffs